presentation-compiler/src/main/scala/org/alephium/ralph/lsp/pc/search/cache/SearchCache.scala

```diff
@@ -9,7 +9,17 @@ import org.alephium.ralph.lsp.utils.WeakHashMapBase
 
 import java.util
 
-case class SearchCache() extends WeakHashMapBase[WorkspaceState.IsSourceAware, WorkspaceSearchCache](new util.WeakHashMap(1, 1)) {
+/**
+ * A cache that stores [[WorkspaceState]] related search information.
+ *
+ * The [[util.WeakHashMap]] is configured to `initialCapacity` and `loadFactor` both set to `1` because
+ * workspace are not expected to often and dynamically.
```

Comment:
missing `change` no?

Reply:
Damnit! Forgot to push the commit for this lol.